### PR TITLE
Fb1161

### PR DESCRIPTION
**Pull request by nicola-spire**

### CODE REVIEW

File: Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx [hunk 1]
```diff
@@ -11,8 +11,20 @@ interface Properties {
   /** The model to display. */
   model: DashboardModel;
 
-  /** The action to perform when logging out. */
-  onLogout?: () => void;
+  /** The action to perform when navigating to the profile. */
```
Comment (kranar) [inline Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx]: The dashboard page internally handles all of these actions, so no prop should be exposed for these. The only action the dashboard page can not possibly handle is the log out, which is why that has its own prop.

File: Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx [hunk 2]
```diff
@@ -28,7 +40,15 @@ export class DashboardPage extends React.Component<Properties, State> {
       isSideMenuOpen: false,
       isLoading: false
     };
-    this.toggleSideMenuIsOpen = this.toggleSideMenuIsOpen.bind(this);
+    this.onToggleMenuIsOpen = this.onToggleMenuIsOpen.bind(this);
```
Comment (kranar) [inline Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx]: This is a better name, thanks.

File: Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx [hunk 3]
```diff
@@ -28,7 +40,15 @@ export class DashboardPage extends React.Component<Properties, State> {
...
+  }
+
+  public static defaultProps = {
```
Comment (kranar) [inline Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx]: Put public static members first (above the constructor).

**Comment by nicola-spire** [comment]
Done!

**Pull request merged**